src/mount-support.c

```diff
@@ -318,8 +317,7 @@ void setup_bind_mounts(const char *appname)
 		return;
 	// however any other error is a real error
 	if (f == NULL) {
-		fprintf(stderr, "Can not open %s (%s)\n", profile_path,
-			strerror(errno));
+                fprintf(stderr, "cannot open %s\n", profile_path);
```

Comment:
I meat that die() does fprintf anyway so you can just do the die thing I listed and it will do the rest